internal/compiler/widgets/fluent/std-widgets.slint

```diff
@@ -308,6 +308,10 @@ export TabImpl := Rectangle {
     horizontal-stretch: 0;
     vertical-stretch: 0;
 
+    border-radius: 2px;
+    border-width: fs.has-focus ? 3px : 0px;
+    border-color: Palette.themeSecondary;
```

Comment:
Did you look at the fluent design from Microsoft for that?

Reply:
No, I just did the easiest thing possible to figure out where the focus went :-)

Reply:
this is what i used for the sesign IIRC.  https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot
sadly there is no focus in there.
How does it look with that change?